jfrog-cli/artifactory/utils/buildinfoproperties_test.go

```diff
@@ -134,3 +166,32 @@ func compareViperConfigs(t *testing.T, actual, expected *viper.Viper, buildType
 		}
 	}
 }
+
+func TestSetProxyIfNeeded(t *testing.T) {
```

Comment:
Question: Isn't this test sufficient for testing the proxy functionality? If it is, can we remove the new additions to the other tests in this file?

Reply:
I think that this only test the setProxyIfDefined function. The other tests testing the entire viper that the information is written correctly in the form of artifactory.proxy.host and artifactory.proxy.port. Those are the values that the build info needs so the other tests purpose is to check that those are the correct keys.